src/SourceGenerators/Uno.UI.SourceGenerators.Internal/DependencyObject/DependencyPropertyGenerator.cs

```diff
@@ -5,527 +5,498 @@
 using System.Globalization;
 using System.Linq;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Uno.Extensions;
 using Uno.UI.SourceGenerators.Helpers;
+using Uno.UI.SourceGenerators.Internal.Extensions;
 using Uno.UI.SourceGenerators.XamlGenerator;
 
 namespace Uno.UI.SourceGenerators.DependencyObject
 {
+	// TODO: We are not respecting IsValidPlatform now.
```

Comment:
Maybe there is no need to do this check. Things are passing and working very well.

I think the reason is that now `Uno.UI.SourceGenerators.Internal` included in less projects, as opposed to when this generator was in Uno.UI.SourceGenerators project which run on much more projects.

Reply:
Indeed. For this particular generator, it was only needed to avoid running it when the requested attribute was not present. Inside of Uno, it's always available.

Reply:
```suggestion
```